wdio/specs/homepage.js

```diff
@@ -5,6 +5,26 @@ describe('Municipality homepage', () => {
     browser.url('/municipality-1/node/1?language=he');
   })
 
+  const checkSelectedLanguage = (language) => {
+    browser.waitForVisible('.btn-group.languages');
+    const activeLanguage= browser.getText('.btn-group.languages .btn.btn-default.active');
+    assert((language == activeLanguage), `Expected page language is ${language}, but instead it's ${activeLanguage}`);
```

Comment:
I think you can remove the `()` from around `language == activeLanguage` (here and below)